.gitmodules

```diff
@@ -11,5 +11,5 @@
 	branch = agave-v2.0
 [submodule "impl/solfuzz"]
 	path = impl/solfuzz
-	url = https://github.com/firedancer-io/solfuzz
+	url = git@github.com:firedancer-io/solfuzz.git
```

Comment:
I don't think we need this

Reply:
Given https://github.com/firedancer-io/solana-conformance/pull/51 do we want it? I don't care either way

Reply:
Do we even need the submodule? What uses it?

Reply:
@topointon-jump

Reply:
we can remove it I think

Reply:
It was added in https://github.com/firedancer-io/solana-conformance/pull/52

The motivation is to make the fuzzing pipeline easier for those creating crashes with solfuzz and those of us reproducing the mismatches.